AllReadyApp/Web-App/AllReady/Areas/Admin/Views/Itinerary/Details.cshtml

```diff
@@ -127,36 +127,24 @@
                 </div>
             }
             <table class="table" style="display: none;" data-bind="visible: hasTeamMembers">
-                <thead>
-                    <tr>
-                        <th>
-                            Name
-                        </th>
-                        <th>
-                            Email
-                        </th>
-                        <th>
-                            Assigned Task
-                        </th>
-                        <th></th>
-                    </tr>
-                </thead>
-                <tbody data-bind="foreach: teamMembers">
+                <tr>
+                    <th>Name</th>
+                    <th>Email</th>
+                    <th>Assigned Task</th>
+                    <th></th>
+                </tr>
+                @foreach(var tm in Model.TeamMembers)
+                {
+                    string style = tm.DisplayName.Contains("Team Lead") ? "font-weight:bold" : null;
```

Comment:
Recommend we add an IsTeamLead property on the TeamMembers in this view model and load it appropriately